src/MVVM/Factories/ViewModelFactory.cs

```diff
@@ -122,18 +121,18 @@ internal static INavigationService TryResolveNavigationServiceInstance()
 
 
         /// <summary>
-        /// Creates a new ViewModel instance, invokes the InitializeAsync method using the parameter and returns the initialized instance
+        /// Creates a new ViewModel instance and invokes the <see cref="IViewModel{TModel}.InitializeAsync(TModel)"/> method.
         /// </summary>
-        /// <typeparam name="TViewModelInterface">Type of the ViewModel to create</typeparam>
-        /// <typeparam name="TModel">Type of the parameter that will be used for initialization</typeparam>
-        /// <returns>ViewModel instance of the given type</returns>
-        public static async Task<TViewModelInterface> ResolveAsync<TViewModelInterface, TModel>(
+        /// <typeparam name="TViewModel">Type of the ViewModel to create</typeparam>
+        /// <typeparam name="TModel">Type of the parameter that will be passed in the <see cref="IViewModel{TModel}.InitializeAsync(TModel)"/> method</typeparam>
```

Comment:
Codacy found an issue: [Split this 161 characters long line (which is greater than 130 authorized).](https://app.codacy.com/gh/UltimateCodeMonkeys/CodeMonkeysMVVM/pullRequest?prid=5767060)